OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDatesPageFragment.kt

```diff
@@ -328,12 +330,6 @@ class CourseDatesPageFragment : OfflineSupportBaseFragment(), BaseFragment.Permi
 
     private fun addOrUpdateEventsInCalendar(calendarId: Long, updateEvents: Boolean) {
```

Comment:
better to rename the method, `addEventsInCalendar`.

Reply:
we are using this method for update calendar snakbar as well so that's why I didn't update them